.travis.yml

```diff
@@ -39,7 +39,7 @@ jobs:
 
 before_install: test/ci/before_install
 
-install: false
+install: skip 
```

Comment:
looks like `install: false` will instruct the machine to actually run `false` instead of skipping the install phase.

Reply:
```
$ false
The command "false" failed and exited with 1 during .
```